dora/core/client/fs/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -24,6 +24,7 @@
  */
 @PublicApi
 @ThreadSafe
+// TODO(jiacheng): this class should just be renamed!
 public final class BlockWorkerInfo {
```

Comment:
I find this class redundant as we already have a `alluxio.wire.WorkerInfo`. We are actively converting `WorkerInfo` to `BlockWorkerInfo` in `FileSystemContext.getAllWorkers` which is unnecessary.